usr/sbin/registercloudguest

```diff
@@ -143,8 +143,38 @@ def setup_registry(registration_target, clean='registry'):
     user, password = utils.get_credentials(
         utils.get_credentials_file(registration_target)
     )
+    with_docker = True
+    if not utils.is_docker_present():
+        with_docker = False
+        logging.info(
+            'Registration for docker registry skipped, '
+            'docker is not installed.'
```

Comment:
do we know that ? we know the file is not present

can it be that docker is installed and the user removed that file ?

Reply:
The message will be the hardest part :) I tried to adapt it